### ISSUE TITLE
Set default alignments in theme.json

### ISSUE THREAD

**Issue by carolinan** [WordPress/gutenberg#44007]
## What problem does this address?

In block themes, when alignments are set in the block markup itself, there is no way to change this alignment using style variations.

Example:
```
<!-- wp:heading {"textAlign":"center","align":"wide","style":{"spacing":{"margin":{"top":"var:preset|spacing|60","bottom":"var:preset|spacing|50"}}}} -->
<h2 class="alignwide has-text-align-center" style="margin-top:var(--wp--preset--spacing--60);margin-bottom:var(--wp--preset--spacing--50)">Mindblown: a blog about philosophy.</h2>
<!-- /wp:heading -->
```

Because `"textAlign":"center"` and `has-text-align-center` is in the markup, it can not be changed by adding a style variation.
If the `textAlign: "center"` was in theme.json, then blue.json could set it to `textAlign: "none"` or similar.

## What is your proposed solution?

Allow setting default alignments for blocks using theme.json. Then alignments that are consistent with the default, can be removed from the HTML files in block themes.

**Comment by mtias** [comment]
Duplicate of #30761 ?

**Comment by carolinan** [comment]
I meant left/right/center alignments.
For example, setting the post title to be centered by default. This was proposed by @critterverse for the style variations in TT3, but it is not possible today.

**Comment by rexwreyes** [comment]
Yes, this was a seriously stupid decision.  I spend more time now trying to figure out how to hack Wordpress because they keep locking you out of setting up different alignments for different screen sizes

**Comment by tyxla** [comment]
I'm closing this one as resolved. 

#59531 added the `textAlign` block support, so theme.json can set `typography.textAlign` globally and per block. 

From the highlighted examples above, Heading (#74383) and Post Title (#75629) have since moved to it, so a style variation can now change their default alignment.

@carolinan, please feel free to reopen or open a new issue if you had something else in mind.
